src/redist/redist.csproj

```diff
@@ -200,41 +198,72 @@
           Condition=" '$(DISABLE_CROSSGEN)' == '' "
           AfterTargets="PublishMSBuildExtensions">
     <ItemGroup>
+      <RoslynFiles Include="$(PublishDir)Roslyn\bincore\**\*" />
```

Comment:
This variable is not for publishing - it's only for crossgen. The variable for what files to include is called `RoslynBits`, and is the first diff above this one. This variable is used for what I describe in the PR as "crossgen batches".

> The way Roslyn handles this is having its Build.dll be in the root Roslyn folder, and then the out-of-proc compiler stuff is in the bincore folder - so it's simple to just crossgen the bincore folder as a batch, and include the Roslyn dir in the "everything else" batch.

(Build.dll, as well as targets, etc.)